<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrainerMenu</class>
 <widget class="QFrame" name="TrainerMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>737</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="5" colspan="2">
    <widget class="QFrame" name="teambuilderFrame">
     <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,1,0,0,0">
      <item>
       <widget class="QClickLabel" name="teambuilderLabel">
        <property name="text">
         <string>Team Builder</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="PokeButtonsHolder" name="pokemonButtons" native="true"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="teamNameLabel">
            <property name="text">
             <string>&amp;Team name</string>
            </property>
            <property name="buddy">
             <cstring>teamName</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <property name="spacing">
             <number>2</number>
            </property>
            <item>
             <widget class="QLineEdit" name="teamName">
              <property name="placeholderText">
               <string>Team name</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="teamTierLabel">
            <property name="text">
             <string>Team tier</string>
            </property>
            <property name="buddy">
             <cstring>teamTier</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="teamTier">
            <property name="maxLength">
             <number>100</number>
            </property>
            <property name="placeholderText">
             <string>Team default tier</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="2" column="0">
           <widget class="QToolButton" name="teamFolderButton">
            <property name="toolTip">
             <string>Change team folder</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonIconOnly</enum>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
            <property name="arrowType">
             <enum>Qt::NoArrow</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QToolButton" name="loadTeam">
            <property name="toolTip">
             <string>Load team</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonIconOnly</enum>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QToolButton" name="saveTeam">
            <property name="toolTip">
             <string>Save team</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonIconOnly</enum>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QToolButton" name="importTeam">
            <property name="toolTip">
             <string>Import team</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonIconOnly</enum>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4"/>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,1,1,0">
        <property name="spacing">
         <number>4</number>
        </property>
        <item row="0" column="0">
         <widget class="QPushButton" name="team1">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="team2">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="team4">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="team3">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="team5">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="team6">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QToolButton" name="addTeam">
          <property name="toolTip">
           <string>Add team</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonIconOnly</enum>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QToolButton" name="removeTeam">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Remove team</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonIconOnly</enum>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" rowspan="4">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="trainerNameLabel">
       <property name="text">
        <string>Trainer &amp;Name</string>
       </property>
       <property name="buddy">
        <cstring>name</cstring>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="name"/>
       </item>
       <item>
        <widget class="QPushButton" name="colorButton">
         <property name="text">
          <string>C&amp;olor</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="informationsLabel">
       <property name="text">
        <string>&amp;Information</string>
       </property>
       <property name="buddy">
        <cstring>infos</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="infos">
       <property name="tabChangesFocus">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="infoLength">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>0/500</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="winningMessageLabel">
       <property name="text">
        <string>&amp;Winning Message</string>
       </property>
       <property name="buddy">
        <cstring>winningMessage</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="winningMessage">
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>200</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="losingMessageLabel">
       <property name="text">
        <string>&amp;Losing Message</string>
       </property>
       <property name="buddy">
        <cstring>losingMessage</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="losingMessage">
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>200</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="tieLabel">
       <property name="text">
        <string>Tie message</string>
       </property>
       <property name="buddy">
        <cstring>tieMessage</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="tieMessage">
       <property name="maxLength">
        <number>200</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0,0,0">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="avatarLabel">
           <property name="text">
            <string>Avatar</string>
           </property>
           <property name="buddy">
            <cstring>avatarNumber</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="avatar">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>96</width>
             <height>96</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>96</width>
             <height>96</height>
            </size>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="topMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QSpinBox" name="avatarNumber">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>729</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="avatarSelect">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QComboBox" name="profileList"/>
         </item>
         <item>
          <widget class="QPushButton" name="saveProfile">
           <property name="text">
            <string>&amp;Save Profile</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="newProfile">
           <property name="text">
            <string>Clea&amp;r Profile</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="deleteProfile">
           <property name="text">
            <string>&amp;Delete Profile</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QPushButton" name="boxCenter">
           <property name="text">
            <string>&amp;Box Center</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="damageCalc">
           <property name="text">
            <string>Damage Calc</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pokedex">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Pokedex</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="6">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Maximum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>60</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="6">
    <widget class="QPushButton" name="close">
     <property name="text">
      <string>&amp;Close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QClickLabel</class>
   <extends>QLabel</extends>
   <header location="global">Utilities/qclicklabel.h</header>
  </customwidget>
  <customwidget>
   <class>PokeButtonsHolder</class>
   <extends>QWidget</extends>
   <header>Teambuilder/pokebuttonsholder.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>name</tabstop>
  <tabstop>colorButton</tabstop>
  <tabstop>infos</tabstop>
  <tabstop>winningMessage</tabstop>
  <tabstop>losingMessage</tabstop>
  <tabstop>tieMessage</tabstop>
  <tabstop>profileList</tabstop>
  <tabstop>saveProfile</tabstop>
  <tabstop>newProfile</tabstop>
  <tabstop>deleteProfile</tabstop>
  <tabstop>boxCenter</tabstop>
  <tabstop>pokedex</tabstop>
  <tabstop>teamName</tabstop>
  <tabstop>teamTier</tabstop>
  <tabstop>loadTeam</tabstop>
  <tabstop>saveTeam</tabstop>
  <tabstop>teamFolderButton</tabstop>
  <tabstop>importTeam</tabstop>
  <tabstop>team1</tabstop>
  <tabstop>team2</tabstop>
  <tabstop>team4</tabstop>
  <tabstop>team3</tabstop>
  <tabstop>team5</tabstop>
  <tabstop>team6</tabstop>
  <tabstop>addTeam</tabstop>
  <tabstop>removeTeam</tabstop>
  <tabstop>close</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>avatarNumber</sender>
   <signal>valueChanged(int)</signal>
   <receiver>TrainerMenu</receiver>
   <slot>setAvatarPixmap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>510</y>
    </hint>
    <hint type="destinationlabel">
     <x>106</x>
     <y>500</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setAvatarPixmap()</slot>
 </slots>
</ui>
